skript.sql

```diff
@@ -0,0 +1,89 @@
+-- MySQL Script generated by MySQL Workbench
+-- Thu May 22 08:57:24 2025
+-- Model: New Model    Version: 1.0
+-- MySQL Workbench Forward Engineering
+
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
+SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0;
+SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_ENGINE_SUBSTITUTION';
+
+-- -----------------------------------------------------
+-- Schema projekt
+-- -----------------------------------------------------
+
+-- -----------------------------------------------------
+-- Schema projekt
+-- -----------------------------------------------------
+CREATE SCHEMA IF NOT EXISTS `projekt` DEFAULT CHARACTER SET utf8 ;
+USE `projekt` ;
+
+-- -----------------------------------------------------
+-- Table `projekt`.`users`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `projekt`.`users` (
+  `id` INT NOT NULL AUTO_INCREMENT,
+  `username` VARCHAR(45) NOT NULL,
+  `password` VARCHAR(100) NOT NULL,
+  `admin` TINYINT NOT NULL,
+  PRIMARY KEY (`id`))
+ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `projekt`.`guns`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `projekt`.`guns` (
+  `id` INT NOT NULL AUTO_INCREMENT,
+  `type` VARCHAR(45) NOT NULL,
+  `Name` VARCHAR(45) NOT NULL,
+  `img` VARCHAR(45) NOT NULL,
+  `Price` VARCHAR(45) NOT NULL,
```

Comment:
Same question about the first letter here. Especially when the second table has all column names in lowercase.